Set m_flow_small to final

mwetter opened the pull request:
This closes #1362.

The entries for the conversion script are
```
convertModifiers("IBPSA.Airflow.Multizone.BaseClasses.PowerLawResistance", {"m_flow_small"}, fill("",0), true);
convertModifiers("IBPSA.Airflow.Multizone.BaseClasses.TwoWayFlowElement", {"m1_flow_small", "m2_flow_small"}, fill("",0), true);
convertModifiers("IBPSA.Airflow.Multizone.EffectiveAirLeakageArea", {"m_flow_small"}, fill("",0), true);
convertModifiers("IBPSA.Airflow.Multizone.Orifice", {"m_flow_small"}, fill("",0), true);
convertModifiers("IBPSA.Airflow.Multizone.DoorDiscretizedOpen", {"m1_flow_small", "m2_flow_small"}, fill("",0), true);
convertModifiers("IBPSA.Airflow.Multizone.DoorDiscretizedOperable", {"m1_flow_small", "m2_flow_small"}, fill("",0), true);
convertModifiers("IBPSA.Airflow.Multizone.ZonalFlow_ACS", {"m1_flow_small", "m2_flow_small"}, fill("",0), true);
convertModifiers("IBPSA.Airflow.Multizone.ZonalFlow_m_flow", {"m1_flow_small", "m2_flow_small"}, fill("",0), true);
```

mwetter commented:
@Mathadon @nytschgeusen @PMehrfeld This change is backward incompatible, although I would not expect a user setting the parameters that are now final (so they don't show up in a gui) because these are not used in the models.
As it is backward incompatible, please each review and approve or let me know requested changes so we can merge it.

PMehrfeld commented:
Although I approved, I have two questions:

- Isn't it necessary to place a new conversion script to [IBPSA/Resources/Scripts/Dymola](https://github.com/ibpsa/modelica-ibpsa/tree/issue_1362_m_flow_small/IBPSA/Resources/Scripts/Dymola)?
- The models `MediumColumn` and `MediumColumnDynamic` do not have a parameter `m_flow_small` but they are mentioned in the first [comment](https://github.com/ibpsa/modelica-ibpsa/pull/1363#issue-417026751).

mwetter commented:
@PMehrfeld : Thanks for the feedback. I updated the conversion script through https://github.com/ibpsa/modelica-ibpsa/pull/1363/commits/7768ad627a5c374dd4ef46598032ca5f1902ea18, and corrected the comment on top of this pull request to remove the medium columns.

nytschgeusen commented:
@mwetter: These changes are ok. for our BuildingSystems library ...
